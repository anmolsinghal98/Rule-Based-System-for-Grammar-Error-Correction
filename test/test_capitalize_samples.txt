I have to go to the mall.
It will be so nice to go to the beach. I want to confirm on which day we are leaving! Are we planning to get there now? I can't wait to spend some time with you. Thanks!
Thank you for your e - mail. I am interested in the soccer club and the tennis club. And I prefer Receiving news by the letter.
it will be so nice to go to the beach! I just want to confirm that on monday or other day. I can't wait to spend some time with you. Outside in the sun.
think that we should get power of communication but also we have a point which is better than each other. Robots doesn't have emotion. So they can't communicate truthwith each other. People getting communication is the best in this world. This is truth. And we have to become the person who can't instead of each other But common is good. Ineccessary for us to there are both in the social They are very difficult. So it is important to make an effort. So let's do.
i think I an most interested in Soccer club topics. I was so glad when I received it.
I'm interested in watching news about sports, for example, soccer, tennis, And so on. I prefer receiving the news by mail on my phone.
i think I am interested in soccer- related topics. I have two reasons. First, Japanese soccer team is very strange. Second, I like watching the soccer game. I used to watch a game of soccer everyday after school in canada.
The director of the who would like to brief the media about Covid-19 situation in europe.
I am going to talk to mr wells about the situation in the parliament.
I disagree with putting manga in the library. I have three reasons to support my opinion. First, you want to study at the library, but manga is so fun to read, so you can't study when you read manga. Second, put the manga in the library and people will come to library and make noise. so, some people can't study. Third, manga is 9 % art. So, you can't think anything. That's why I disagree with putting manga in the library.
I was listening to the new Run the Jewels album. It is mostly hardcore hip hop and I'd heard about it through Youtube reviews online. Fantano scored it an 8.
I was listening to the new Run the jewels album. It is mostly hardcore Hip hop and iâ€™d heard about it through youtube reviews online. Fantano scored it an 8.
I was going to the park to hang out with my friends. My friends are in the city and I'm currently busy with work and can't talk right now.
I was going to the Park to hang out with my friends. My friends are in the city and i'm currently busy with work and can't talk right now.
My friend asked me to recommend a laptop to them since they were looking for one. I recommended Toshiba since it's a good company.
My friend asked me to recommend a laptop to them since they were looking for one. I recommended toshiba since It's a good company.
I have a friend who lives in Japan and their name is Sakura. I will meet them today.
I have a friend who lives in japan and their name is sakura. I will meet them Today.
When we were kids, we used to love playing games on our Nintendo.
When we were kids, we used to love playing games on our nintendo.