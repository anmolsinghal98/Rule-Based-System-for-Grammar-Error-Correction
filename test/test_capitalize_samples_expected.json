[
  {
    "Sample No": 1,
    "Detections": []
  },
  {
    "Sample No": 2,
    "Detections": []
  },
  {
    "Sample No": 3,
    "Detections": [
      {
        "Sentence ID": 2,
        "Word Index": "4",
        "Word String": "Receiving",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 4,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "1",
        "Word String": "it",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 5,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "1",
        "Word String": "think",
        "Error Type": "Capitalization"
      },
      {
        "Sentence ID": 5,
        "Word Index": "15",
        "Word String": "But",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 6,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "1",
        "Word String": "i",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 7,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "17",
        "Word String": "And",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 8,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "1",
        "Word String": "i",
        "Error Type": "Capitalization"
      },
      {
        "Sentence ID": 2,
        "Word Index": "3",
        "Word String": "Japanese",
        "Error Type": "Capitalization"
      },
      {
        "Sentence ID": 4,
        "Word Index": "13",
        "Word String": "canada",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 9,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "18",
        "Word String": "europe",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 10,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "7",
        "Word String": "mr",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 11,
    "Detections": [
      {
        "Sentence ID": 4,
        "Word Index": "1",
        "Word String": "so",
        "Error Type": "Capitalization"
      },
      {
        "Sentence ID": 5,
        "Word Index": "3",
        "Word String": "manga",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 12,
    "Detections": []
  },
  {
    "Sample No": 13,
    "Detections": [
      {
        "Sentence ID": 1,
        "Word Index": "8",
        "Word String": "i",
        "Error Type": "Capitalization"
      },
      {
        "Sentence ID": 1,
        "Word Index": "14",
        "Word String": "youtube",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 14,
    "Detections": []
  },
  {
    "Sample No": 15,
    "Detections": [
      {
        "Sentence ID": 1,
        "Word Index": "8",
        "Word String": "i",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 16,
    "Detections": []
  },
  {
    "Sample No": 17,
    "Detections": [
      {
        "Sentence ID": 1,
        "Word Index": "3",
        "Word String": "toshiba",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 18,
    "Detections": []
  },
  {
    "Sample No": 19,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "8",
        "Word String": "japan",
        "Error Type": "Capitalization"
      },
      {
        "Sentence ID": 0,
        "Word Index": "13",
        "Word String": "sakura",
        "Error Type": "Capitalization"
      },
      {
        "Sentence ID": 1,
        "Word Index": "5",
        "Word String": "Today",
        "Error Type": "Capitalization"
      }
    ]
  },
  {
    "Sample No": 20,
    "Detections": []
  },
  {
    "Sample No": 21,
    "Detections": [
      {
        "Sentence ID": 0,
        "Word Index": "14",
        "Word String": "nintendo",
        "Error Type": "Capitalization"
      }
    ]
  }
]